stable/velero/values.yaml

```diff
@@ -5,8 +5,8 @@
 # Details of the container image to use in the Velero deployment & daemonset (if
 # enabling restic). Required.
 image:
-  repository: gcr.io/heptio-images/velero
-  tag: v1.1.0
+  repository: velero/velero
+  tag: v1.2.0-beta.1
```

Comment:
Can we hold this change back for now? There's this PR that is on hold that we can use to update to 1.2.0 and add support for cross-subscription backups in azure https://github.com/helm/charts/pull/17585/files